### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
Updated EN -> JA documentation translations

English sentences exist in this PR. Please leave them as they are. (They will be translated next time.)

### CODE REVIEW

**Review by catherinepope — APPROVED** [review]
👍

File: jekyll/_cci2_ja/artifacts.md [hunk 1]
```diff
@@ -32,7 +32,9 @@ CircleCI Web アプリでパイプラインの **Job** ページに移動し、[
 
 ![[Artifacts (アーティファクト)] タブのスクリーンショット]({{site.baseurl}}/assets/img/docs/artifacts.png)
 
-**アーティファクトへは作成から30日間アクセスできます。 **  ドキュメントや永続的なコンテンツのソースとして依存している場合、S3や静的Webサイト用のGitHub Pages、Netlifyのような専用領域にデプロイすることを推奨します。
+デフォルトのアーティファクトの保存期間は 30 日間です。 保存期間は、[CircleCI Web アプリ](https://app.circleci.com/)の **Plan > Usage Controls** からカスタマイズ可能です。 現在、設定できる保存期間の最大値が 30 日間となっています。
+
+ネットワークとストレージ使用量の管理の詳細については、[データの永続化]({{site.baseurl}}/2.0/persist-data/)のページを参照してください。
```
[the remark this answers is not in the document]
Comment (nanophate) [reply jekyll/_cci2_ja/artifacts.md:37]: Seems like adding `ja` is necessary looking at the following translated section

https://github.com/circleci/circleci-docs/pull/6639/files#diff-5bd8ed22601414f269de7b4f3fb9fa94e775f623d78c59147d903de6964f9304R301

File: jekyll/_cci2_ja/aws.adoc [hunk 2]
```diff
@@ -211,25 +211,25 @@ NOTE: 「Unknown error authenticating via GitHub. Try again, or contact us.」
 
 .. GitHub から Client ID と Secret をコピーして、該当するフィールドにペーストし、[Test Authentication] をクリックします。
 
-.. GitHub.com をご利用の場合は、ステップ 6 に進みます。 Github Enterprise をご利用の場合は、いくつかの追加手順を行い、API トークンを提供していただき、お客様の組織を確認する必要があります。 トークンを提供するには、GitHub Enterprise のダッシュボードから以下を実行してください。
+.. GitHub.com をご利用の場合は、ステップ 6 に進みます。 If using Github Enterprise, you will also need to follow some supplementary steps and supply an API Token so we can verify your organization. トークンを提供するには、GitHub Enterprise のダッシュボードから以下を実行してください。
```
Comment (nanophate) [inline jekyll/_cci2_ja/aws.adoc:214]: 👀 Translation went to original language

File: jekyll/_cci2_ja/aws.adoc [hunk 3]
```diff
@@ -211,25 +211,25 @@ NOTE: 「Unknown error authenticating via GitHub. Try again, or contact us.」
...
 ... Personal Settings (右上) に行き、  Developer Settings  Personal Access Tokens に移動します。
 ... [generate new token] をクリックします。 誤って削除されないように、トークンには適切な名前を付けてください。 チェックボックスには何もチェックを入れないでください。ここでは、デフォルトのパブリックな読み取りレベルのアクセスが必要なだけで、追加のアクセス権限は必要ありません。 このトークンは一人のユーザーが所有するのではなく、組織全体で共有することをお勧めします。
 ... 新しいトークンをコピーして、GitHub Enterprise Default API Token フィールドに貼り付けます。
 +
 .Github Enterprise Token を入力します。
 image::ghe_token.png[Github Integration]
 
-. **LDAP**: インストールに LDAP 認証を使用する場合は、LDAP のセクションに必要な情報を入力してください。 LDAP 設定の詳細については、 https://circleci.com/docs/2.0/authentication/#ldap[LDAP 認証ガイド]をご覧ください。
+. **LDAP**: インストールに LDAP 認証を使用する場合は、LDAP のセクションに必要な情報を入力してください。 For a detailed run-through of LDAP settings, read our https://circleci.com/docs/2.0/authentication/#ldap[LDAP authentication guide]
```
Comment (nanophate) [inline jekyll/_cci2_ja/aws.adoc:222]: 👀 Translation went to original language

File: jekyll/_cci2_ja/aws.adoc [hunk 4]
```diff
@@ -265,10 +265,10 @@ NOTE: 管理コンソールで、 `Failure reported from operator: no such image
...
-image::dashboard.png[]
+image::dashboard.png[CircleCI server dashboard]
 
 . ビルドコンテナが起動してイメージがダウンロードされると、すぐに最初のビルドを開始します。 **約 15 分**が経過し、[Refresh] ボタンをクリックしても更新が行われない場合は、 https://support.circleci.com/hc/ja[CircleCI サポート]にお問い合わせください。
 
 . 次に、 https://github.com/circleci/realitycheck[realitycheck レポジトリ]を使用して、基本的な CircleCI 機能を確認します。
 
-. 最初のビルドの実行に失敗する場合は、まず https://circleci.com/docs/2.0/troubleshooting[トラブルシューティング]ガイドで一般的なトラブルシューティングのトピックを参照してください。CircleCI Server 内の Builder の状態を確認する方法については、 https://circleci.com/docs/2.0/nomad[Nomad クラスタの操作ガイド]を参照してください。
\ No newline at end of file
+. 最初のビルドの実行に失敗する場合は、まず https://circleci.com/docs/2.0/troubleshooting[トラブルシューティング]ガイドで一般的なトラブルシューティングのトピックを参照してください。CircleCI Server 内の Builder の状態を確認する方法については、 https://circleci.com/docs/2.0/nomad[Nomad クラスタの操作ガイド]を参照してください。
```
Comment (nanophate) [inline jekyll/_cci2_ja/aws.adoc:274]: Adding `/ja` for link might be nice for customer.

- https://circleci.com/docs/ja/2.0/troubleshooting
- https://circleci.com/docs/ja/2.0/nomad

File: jekyll/_cci2_ja/caching-strategy.md [hunk 5]
```diff
@@ -16,6 +16,15 @@ version:
...
 {:toc}
 
+## キャッシュストレージのカスタマイズ
+{: #caching-and-self-hosted-runner }
+
+セルフホストランナーを使用する場合、プランに含まれるネットワークとストレージ使用量には制限があります。 キャッシュに関連するアクションには、ネットワークとストレージの使用が発生するものがあります。 お客様の使用量が制限を超えた場合、料金が発生します。
+
+キャッシュを長期間保存すると、ストレージコストに影響が及ぶため、キャッシュを保存する理由やユースケースに応じた必要なキャッシュの保存期間を明確にすることをお勧めします。 ニーズに合う場合は、キャッシュのストレージ保存期間を短く設定し、コストを削減しましょう。
+
+[CircleCI Web アプリ](https://app.circleci.com/)で **Plan > Usage Controls** に移動し、キャッシュのストレージ使用量や保存期間をカスタマイズすることができます。 ネットワークとストレージ使用量の管理の詳細については、[データの永続化]({{site.baseurl}}/2.0/persist-data/)のページを参照してください。
```
Comment (nanophate) [inline jekyll/_cci2_ja/caching-strategy.md:26]: `{{site.baseurl}}/ja/2.0/persist-data/` to add `ja` before `2.0` for better experience.

File: jekyll/_cci2_ja/caching.md [hunk 6]
```diff
@@ -366,22 +363,22 @@ myapp-+KlBebDceJh_zOWQIAJDLEkdkKoeldAldkaKiallQ=
       - image: customimage/ruby:2.3-node-phantomjs-0.0.1
         auth:
           username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # コンテキスト/プロジェクト UI の環境変数を参照
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
```
Comment (nanophate) [inline jekyll/_cci2_ja/caching.md:366]: 👀 Translation went to original language

File: jekyll/_cci2_ja/caching.md [hunk 7]
```diff
@@ -366,22 +363,22 @@ myapp-+KlBebDceJh_zOWQIAJDLEkdkKoeldAldkaKiallQ=
...
         environment:
           RAILS_ENV: test
           RACK_ENV: test
-      - image: circleci/mysql:5.6
+      - image: cimg/mysql:5.7
         auth:
           username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # コンテキスト/プロジェクト UI の環境変数を参照
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
```
Comment (nanophate) [inline jekyll/_cci2_ja/caching.md:373]: 👀 Translation went to original language

File: jekyll/_cci2_ja/caching.md [hunk 8]
```diff
@@ -366,22 +363,22 @@ myapp-+KlBebDceJh_zOWQIAJDLEkdkKoeldAldkaKiallQ=
...
     steps:
       - checkout
       - run: cp config/{database_circleci,database}.yml
 
-      # Bundler の実行
-      # 可能な場合は、インストールされている gem をキャッシュから読み込み、バンドル インストール後にキャッシュを保存します
-      # 複数のキャッシュを使用して、キャッシュ ヒットの確率を上げます
+      # Run bundler
+      # Load installed gems from cache if possible, bundle install then save cache
+      # Multiple caches are used to increase the chance of a cache hit
```
Comment (nanophate) [inline jekyll/_cci2_ja/caching.md:381]: 👀 Translation went to original language

File: jekyll/_cci2_ja/caching.md [hunk 9]
```diff
@@ -400,9 +397,9 @@ myapp-+KlBebDceJh_zOWQIAJDLEkdkKoeldAldkaKiallQ=
       - run: bundle exec rake db:create db:schema:load --trace
       - run: bundle exec rake factory_girl:lint
 
-      # アセットのプリコンパイル
-      # 可能な場合は、アセットをキャッシュから読み込み、アセットのプリコンパイル後にキャッシュを保存します
-      # 複数のキャッシュを使用して、キャッシュ ヒットの確率を上げます
+      # Precompile assets
+      # Load assets from cache if possible, precompile assets then save cache
+      # Multiple caches are used to increase the chance of a cache hit
```
Comment (nanophate) [inline jekyll/_cci2_ja/caching.md:402]: 👀 Translation went to original language

File: jekyll/_cci2_ja/code-coverage.md [hunk 10]
```diff
@@ -556,13 +556,15 @@ jobs:
       - image: cimg/php:8.1-browsers
         auth:
           username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # コンテキスト/プロジェクト UI 環境変数を参照します。 e
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
```
Comment (nanophate) [inline jekyll/_cci2_ja/code-coverage.md:559]: 👀 Translation went to original language

File: jekyll/_cci2_ja/code-coverage.md [hunk 11]
```diff
@@ -578,13 +580,15 @@ jobs:
       - image: cimg/php:8.1-browsers
         auth:
           username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # コンテキスト/プロジェクト UI 環境変数を参照します。
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
```
Comment (nanophate) [inline jekyll/_cci2_ja/code-coverage.md:583]: 👀 Translation went to original language

File: jekyll/_cci2_ja/code-coverage.md [hunk 12]
```diff
@@ -602,12 +606,14 @@ jobs:
       - image: circleci/php:7-fpm-browsers-legacy
         auth:
           username: mydockerhub-user
-          password: $DOCKERHUB_PASSWORD  # コンテキスト/プロジェクト UI 環境変数を参照します。
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
```
Comment (nanophate) [inline jekyll/_cci2_ja/code-coverage.md:609]: 👀 Translation went to original language

**Review by nanophate — APPROVED** [review]
Commented on section where `ja` was missing from link. Other than that it looked fine.

**Comment by Mayuko-Sa** [comment]
Thank you @nanophate !
Regarding missing /ja/ and translation regression, let me fix these in subsequent translations. (Sorry I didn't tell you about this)

**Pull request merged**